lib/node_modules/@stdlib/math/base/special/asinf/README.md

```diff
@@ -0,0 +1,175 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# asinf
+
+> Compute the [arcsine][arcsine] of a single-precision floating-point number.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var asinf = require( '@stdlib/math/base/special/asinf' );
+```
+
+#### asinf( x )
+
+Computes the [arcsine][arcsine] of a single-precision floating-point number (in radians).
+
+```javascript
+var v = asinf( 0.0 );
+// returns 0.0
+
+v = asinf( -3.14/6.0 );
+// returns ~-0.551
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var linspace = require( '@stdlib/array/base/linspace' );
+var asinf = require( '@stdlib/math/base/special/asinf' );
+
+var x = linspace( -1.0, 1.0, 100 );
+
+var i;
+for ( i = 0; i < x.length; i++ ) {
+    console.log( asinf( x[ i ] ) );
+}
+```
+
+</section>
+
+<!-- /.examples -->
+
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/special/asinf.h"
+```
+
+#### stdlib_base_asinf( x )
+
+Computes the [arcsine][arcsine] of a single-precision floating-point number (in radians).
+
+```c
+float out = stdlib_base_asinf( 0.0 );
+// returns 0.0
+
+out = stdlib_base_asinf( -3.14/6.0 );
+// returns ~-0.551
+```
+
+The function accepts the following arguments:
+
+-   **x**: `[in] float` input value (in radians).
+
+```c
+float stdlib_base_asinf( const float x );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/math/base/special/asinf.h"
+#include <stdio.h>
+
+int main( void ) {
+    const float x[] = { -1.0, -0.78, -0.56, -0.33, -0.11, 0.11, 0.33, 0.56, 0.78, 1.0 };
```

Comment:
```suggestion
    const float x[] = { -1.0f, -0.78f, -0.56f, -0.33f, -0.11f, 0.11f, 0.33f, 0.56f, 0.78f, 1.0f };
```

Use suffix to type as single-precision.

Reply:
Thanks for pointing it out, I'll keep that in mind !